test/e2e/kubevirt.go

```diff
@@ -1432,9 +1435,9 @@ chpasswd: { expire: False }`
 				iperfServerPodCondition = checkPodRunningReady()
 			}
 			iperfServerArgs := ""
-			if td.test.description == liveMigrate.description {
-				// Forver shutdown if tcp connection is broken
-				iperfServerArgs = "--one-off"
+			if td.test.description == liveMigrate.description && td.topology == "layer2" /* localnet breaks tcp connection*/ {
```

Comment:
I think it's a lot clearer if we have the localnet tests explicitly marked as not working - `XEntry` / `PEntry` - until we fix the underlying issue. 

Skipping part of the test in the middle of it is hard to follow or reason about. 
Do we know why localnet breaks TCP connection ? Is there an opened issue for it ? It's paramount that we address it ASAP - we're now recommending users to use it instead of bridge binding.

Reply:
I have open a bug for this https://issues.redhat.com/browse/OCPBUGS-44652